File: Klijent/Sysreq.js
export class Sysreq{

    constructor(id, cpu, gpu, ram, freeSpace, igraFK){
        this.id = id;
        this.cpu = cpu;
        this.gpu = gpu;
        this.ram = ram;
        this.freeSpace = freeSpace;
        this.igraFK = igraFK;
    }
    
    crtaj(host)
    {   
        var tr = document.createElement("tr");
        host.appendChild(tr);

        var el = document.createElement("td");
        el.innerHTML=this.cpu;
        tr.appendChild(el);

        el = document.createElement("td");
        el.innerHTML=this.gpu;
        tr.appendChild(el);

        el = document.createElement("td");
        el.innerHTML=this.ram;
        tr.appendChild(el);

        el = document.createElement("td");
        el.innerHTML=this.freeSpace;
        tr.appendChild(el);
    }
}

File: Server/Models/Sysreq.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Models
{
    [Table("Sysreq")]
    public class Sysreq
    {
        [Key]
        public int ID { get; set; }

        [Required]
        [MaxLength(100)]
        public string CPU { get; set; }

        [Required]
        [MaxLength(100)]
        public string GPU { get; set; }

         [Required]
         [Range(1,256)]
        public int RAM { get; set; }
        
        [Required]
        [Range(1,10000)]
        public int FreeSpace { get; set; }

        public Igra IgraFK { get; set; }
    }
}